src/main/java/com/puppycrawl/tools/checkstyle/checks/FinalParametersCheck.java

```diff
@@ -68,6 +68,24 @@
  * &lt;module name=&quot;FinalParameters&quot;/&gt;
  * </pre>
  * <p>
+ * Example :
+ * </p>
+ * <pre>
+ * class MyClass {
+ *   public MyClass(final int num) { // OK
+ *   }
+ *   public void myMethod(final String str) { // OK
+ *   }
+ * }
+ *
+ * class MyClass {
+ *   public MyClass(int num) { // violation, num should be final
```

Comment:
please move this ctor to previous class  and change type of parameter.

code in example/pre tag, should be compile-able. Two classes with same name is not ok.

in this way you will show all on single class.
And it is better to reuse the same class in all other examples to show how behavior is changing by usage of some properties.